services/cases-api/lambdas/updateCase.js

```diff
@@ -179,19 +199,24 @@ function getNewCaseStatus(answers, signature) {
   return getStatusByType(statusType);
 }
 
-function isAnswersEncrypted(answers) {
+function areAnswersEncrypted(answers) {
```

Comment:
like is over are :)
isEncrypted() is maybe enought because you put answers in args?